### PR TITLE
no response page moved to parent dir due to common use.

### PR DESCRIPTION
**Pull request by velimd**
### Change description ###
image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/steps/no-response-yet/content.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { TranslationFn } from '../../app/controller/GetController';
+import { SWITCH_TO_SOLE_APPLICATION } from '../urls';
+
+const en = ({ partner }) => ({
+  title: `Your ${partner} has not yet responded`,
+  line1: `Your ${partner} has not yet responded. The quickest way to progress your application is to contact them and ask them to respond, if it’s safe to do so. They can still respond, even though the deadline has passed.`,
+  readMore: 'If you do not think they will respond',
+  line2: `If you think that your ${partner} will not respond then you can create a new ‘sole application’.`,
+  line3: `This means that you will submit the application on your own, and your ${partner} will respond to it after it has been submitted and checked by the court.`,
+  line4: `You can <a href="${SWITCH_TO_SOLE_APPLICATION}" class="govuk-link">create a new application here.</a>`,
```
Comment (benouaer) [inline src/main/steps/no-response-yet/content.ts:10]: could the anchor tag be moved to the template?

File: src/main/steps/no-response-yet/get.test.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  });
+
+  test('Should render the no response page with civil content', async () => {
+    const req = mockRequest();
+    const res = mockResponse();
+    res.locals.serviceType = DivorceOrDissolution.DISSOLUTION;
+    await controller.get(req, res);
+    const isDivorce = false;
+
+    expect(res.render).toBeCalledWith(expect.anything(), {
```
Comment (benouaer) [inline src/main/steps/no-response-yet/get.test.ts:40]: is there a way to check the actual text displayed on the page?

File: src/main/steps/no-response-yet/get.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { GetController } from '../../app/controller/GetController';
+
+import { generateContent } from './content';
+
+export class NoResponseYetApplicationGetController extends GetController {
+  constructor() {
+    super(__dirname + '/template.njk', generateContent);
```
Comment (benouaer) [inline src/main/steps/no-response-yet/get.ts:7]: not sure what's happening here

**Review by benouaer — APPROVED** [review]

File: src/main/steps/no-response-yet/get.ts (hunk 3 above)
Comment (velimd) [reply src/main/steps/no-response-yet/get.ts:7]: its just setting the controller up with the template location and content function. its either set it here or set it in routes.ts but having to import every content function and set the path for every page that isnt in the sequence file.

File: src/main/steps/no-response-yet/get.test.ts (hunk 2 above)
Comment (velimd) [reply src/main/steps/no-response-yet/get.test.ts:40]: we can match some content

Comment (velimd) [reply src/main/steps/no-response-yet/get.test.ts:40]: the expect anything part is just the path to the template file, do we need to test that?

**Pull request merged**